sdks/urbackend-python/src/urbackend/auth.py

```
@@ -0,0 +1,448 @@
+        Example:
+            >>> client.auth.logout()
+        """
+        active_token = token or self._session_token
+        result: Dict[str, Any] = {"success": True, "message": "Logged out locally"}
+        if active_token:
+            try:
+                result = self._http.request(
+                    "POST", "/api/userAuth/logout", token=active_token
+                )
+            except Exception:
+                pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)